docs/SUMMARY.md

```diff
@@ -1,6 +1,6 @@
 # Summary
 
- * [{{site.data.keyword.edge}} 4.2](kc_welcome_containers.html)
+ * [{{site.data.keyword.edge}} 4.3](kc_welcome_containers.html)
```

Comment:
I'm not sure that "4.3" has any meaning in this context.  Is there are variable we should use instead?

Reply:
There is a variable `{{site.data.keyword.version}}`    It currently is 4.2 in the OH _data/keyword.yml  Which might be outdated?   Not even certain if OH has a concept of version. 

Here's what I'll do. I'll bring in `{{site.data.keyword.version}}` via another commit to this PR.  Then, separately, we can decide what "version" means for OH.

Reply:
Resolved by https://github.com/open-horizon/open-horizon.github.io/pull/220/commits/04aefdbc315babe5e1cec5ec1ec1b9e34f4c8a40